src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import PeopleTable from './PeopleTable';
+import ErrorMessage from './ErrorMessage';
+import NotPeople from './NotPeople';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    let isMounted = true;
+
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        if (isMounted) {
+          setPeople(data);
+        }
+      })
+      .catch(() => {
+        if (isMounted) {
+          setIsError(true);
+        }
+      })
+      .finally(() => {
+        if (isMounted) {
+          setIsLoading(false);
+        }
+      });
+
+    return () => {
+      isMounted = false;
+    };
```

Comment:
The 'isMounted' variable is used to avoid setting the state on an unmounted component, which is good. However, this pattern is not recommended in the latest React versions. Consider using an AbortController to cancel the request when the component unmounts instead.